agent/main.go

```diff
@@ -256,13 +252,14 @@ func hostApi(socketHarOutputChannel chan<- *tapApi.OutputChannelItem) {
 	app.Use(DisableRootStaticCache())
 
 	if err := setUIMode(); err != nil {
-		logger.Log.Panicf("Error setting ui mode, err: %v", err)
+		logger.Log.Errorf("Error setting ui mode, err: %v", err)
 	}
 	app.Use(static.ServeRoot("/", "./site"))
 
 	app.Use(CORSMiddleware()) // This has to be called after the static middleware, does not work if its called before
 
 	api.WebSocketRoutes(app, &eventHandlers, startTime)
+	routes.StandaloneRoutes(app)
```

Comment:
hostApi is called for basic mizu as well, am i wrong?
if so basic mizu should have these routes?